### ISSUE TITLE
What is the impact if actual cases are several multiples of confirmed cases?

### ISSUE THREAD

**Issue by scherrey** [k-sys/covid-19#21]
Outstanding job on this projection and tracking! This nearly brought tears to my eyes and I literally said "YES!" when I read it because so many people have been abusing this kind of model: 

"Yet, today, we don't yet use $R_t$ in this way. In fact, the only real-time measure I've seen has been for Hong Kong. More importantly, it is not useful to understand $R_t$ at a national level. Instead, to manage this crisis effectively, we need a local (state, county and/or city) granularity of $R_t$."

I've been saying this for weeks! Great to see someone express it with good numbers.

I've been tracking rates of growth for both new cases and deaths. I realized last night and am now absolutely certain that the rates of growth for new cases has been entirely bounded by the number of tests we can conduct per day. National rates of positives have been locked in right at 20%. We're about 700k current confirmed cases today. If I want there to be 1M confirmed cases I just need to conduct 1.5M more tests and I guarantee that I'd have that number. Right now at 150k/day capacity it'll happen in 10 days (effective yesterday). If we doubled our capacity to 300k/day we'd see it in five days. 

What's happened is that, somewhere around March 19, the rates of infection overtook our capacity for daily tests so there's no telling how many actual cases are now out there. Until that is corrected our confirmed cases numbers are absolutely, to use the technical term of art, bogus. The Standford study seems to confirm this opinion. 

Soo.... let's say (for example) actual incidents have grown since March 19 to 10x the confirmed incidents. Does that make Rt higher or lower and by how much? 

Anyway - outstanding job and thanks for the excellent work.

**Comment by kmedved** [comment]
It depends on the multiplier. If the multiplier is constant (10x the whole time), this should have no impact. If, more likely, the multiplier is shrinking (we had a huge increase in testing from ~March 10 to March 26), then it would have a big impact.

**Comment by scherrey** [comment]
The impact would be that the Rt goes lower or higher? We jumped up to 50k tests/day around then then jumped up again to 150k tests/day and have, unfortunately, held steady there.

**Comment by tstirrat15** [comment]
Doesn't this implicitly assume that tests are randomly sampled from the population?

If testing is preferentially given to sick patients, and there are more new infections in a given day than there are tests, won't the effective testing capacity be effectively saturated and indicate a lower $R_t$?

**Comment by kmedved** [comment]
Yes @tstirrat15 - that's why there's a multiplier in the first place (e.g., 10x higher than confirmed cases).

If the multiplier changes to become smaller (i.e., we're catching a larger portion of true cases than before), that would cause Rt to be overestimated by using confirmed case counts.
